src/shared/tpm2-util.c

```diff
@@ -2119,6 +2119,8 @@ int tpm2_create_primary(
                         /* creationData= */ NULL,
                         /* creationHash= */ NULL,
                         /* creationTicket= */ NULL);
+        if (rc == TPM2_RC_BAD_AUTH)
```

Comment:
hmm, i am a bit confused. The error on DA lockout mode, isn't that TPM2_RC_LOCKOUT?

What kind of lock are we actually talking about here?